deliverytiersuite/delivery-tier-suite/metadata/pom.xml

```diff
@@ -26,7 +26,7 @@
         <dependency>
             <groupId>org.liquibase</groupId>
             <artifactId>liquibase-core</artifactId>
-            <version>4.8.0</version>
+            <version>4.23.1</version>
```

Comment:
Can remove version as it is in parent pom.xml

Reply:
Nate liquibase-core is not in parent pom.